Common/src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -90,7 +90,7 @@
   "power.toomanyorigins.cannibalism.name": "Yamyamlık",
   "power.toomanyorigins.cannibalism.description": "Çürük et senin için daha besleyici.",
   "power.toomanyorigins.opposite_day.name": "Ters Gün",
-  "power.toomanyorigins.opposite_day.description": "Anında Hasar ve Anında Sağlık efekti senin üzerinde ters. Ayrıca açlığa ve zombileşmeye dirençlisin.",
+  "power.toomanyorigins.opposite_day.description": "Anında Hasar ve Anında Sağlık etkileri senin üzerinde ters. Ayrıca açlığa ve zombileşmeye dirençlisin.",
```

Comment:
made effect plural and appearently Effect doesn't translate to efekt in the base game but translates to etki, silly oversight by me